Sources/KSCrashRecording/KSCrashReportC.c

```diff
@@ -899,7 +900,10 @@ static void writeNotableStackContents(const KSCrashReportWriter *const writer,
     char nameBuffer[40];
     for (uintptr_t address = lowAddress; address < highAddress; address += sizeof(address)) {
         if (ksmem_copySafely((void *)address, &contentsAsPointer, sizeof(contentsAsPointer))) {
-            sprintf(nameBuffer, "stack@%p", (void *)address);
+            memcpy(nameBuffer, "stack@0x", 8);
+            char *addressStart = nameBuffer + 8;
+            size_t addressLength = kssc_uint64_to_hex((uintptr_t)address, addressStart, 1, false);
+            addressStart[addressLength] = 0;
```

Comment:
Isn't it already null terminated in the function? Or is it done here for some special situation?